scripts/report/report.sh

```diff
@@ -22,61 +22,157 @@ scriptDir=$3
+	parse_to_report "$sta_log" "$REPORT_PATH" opensta_wns.rpt wns_report wns_report_end
+	parse_to_report "$sta_log" "$REPORT_PATH" opensta_tns.rpt tns_report tns_report_end
+	parse_to_report "$sta_log" "$REPORT_PATH" opensta.slew.rpt check_slew check_slew_end
+else
+	echo "Static Timing Analysis log not found or empty." > "$REPORT_PATH/opensta.rpt"
+fi
+
+if [[ $(test_file "$sta_post_resizer_log") ]]; then
+	parse_to_report "$sta_post_resizer_log" "$REPORT_PATH" opensta_post_resizer.timing.rpt timing_report timing_report_end
+	parse_to_report "$sta_post_resizer_log" "$REPORT_PATH" opensta_post_resizer.min_max.rpt min_max_report min_max_report_end
+	parse_to_report "$sta_post_resizer_log" "$REPORT_PATH" opensta_post_resizer.rpt check_report check_report_end
+	parse_to_report "$sta_post_resizer_log" "$REPORT_PATH" opensta_post_resizer_wns.rpt wns_report wns_report_end
+	parse_to_report "$sta_post_resizer_log" "$REPORT_PATH" opensta_post_resizer_tns.rpt tns_report tns_report_end
+	parse_to_report "$sta_post_resizer_log" "$REPORT_PATH" opensta_post_resizer.slew.rpt check_slew check_slew_end
+else
+	echo "Static Timing Analysis Post Resizer log not found or empty." > "$REPORT_PATH/opensta_post_resizer.rpt"
+fi
+
+if [[ $(find "$sta_post_resizer_timing_log" -type f -size +10c 2> /dev/null) ]]; then
+	parse_to_report "$sta_post_resizer_timing_log" "$REPORT_PATH" opensta_post_resizer_timing.timing.rpt timing_report timing_report_end
+	parse_to_report "$sta_post_resizer_timing_log" "$REPORT_PATH" opensta_post_resizer_timing.min_max.rpt min_max_report min_max_report_end
+	parse_to_report "$sta_post_resizer_timing_log" "$REPORT_PATH" opensta_post_resizer_timing.rpt check_report check_report_end
+	parse_to_report "$sta_post_resizer_timing_log" "$REPORT_PATH" opensta_post_resizer_timing_wns.rpt wns_report wns_report_end
+	parse_to_report "$sta_post_resizer_timing_log" "$REPORT_PATH" opensta_post_resizer_timing_tns.rpt tns_report tns_report_end
+	parse_to_report "$sta_post_resizer_timing_log" "$REPORT_PATH" opensta_post_resizer_timing.slew.rpt check_slew check_slew_end
+else
+	echo "Static Timing Analysis Post Resizer Timing log not found or empty." > "$REPORT_PATH/opensta_post_resizer_timing.rpt"
+fi
+
+if [[ $(test_file "$sta_spef_log") ]]; then
+	parse_to_report "$sta_spef_log" "$REPORT_PATH" opensta_spef.timing.rpt timing_report timing_report_end
+	parse_to_report "$sta_spef_log" "$REPORT_PATH" opensta_spef.min_max.rpt min_max_report min_max_report_end
+	parse_to_report "$sta_spef_log" "$REPORT_PATH" opensta_spef.rpt check_report check_report_end
+	parse_to_report "$sta_spef_log" "$REPORT_PATH" opensta_spef_wns.rpt wns_report wns_report_end
+	parse_to_report "$sta_spef_log" "$REPORT_PATH" opensta_spef_tns.rpt tns_report tns_report_end
+	parse_to_report "$sta_spef_log" "$REPORT_PATH" opensta_spef.slew.rpt check_slew check_slew_end
+else
+	echo "Static Timing Analysis SPEF log not found or empty." > "$REPORT_PATH/opensta_spef.rpt"
+fi
+
+
```

Comment:
Place separate reports writing before the final summary since the latter expects the presence of the first ones